argocd/base/kustomization.yaml

```diff
@@ -13,7 +14,7 @@ patchesStrategicMerge:
 images:
 - name: quay.io/argoproj/argocd
   newName: quay.io/operate-first/argocd
-  newTag: v2.0.1-1
+  newTag: v2.1.3-1
 - name: redis
   newName: quay.io/operate-first/redis
-  newTag: 6.2.1-alpine
+  newTag: 6.2.4-alpine
```

Comment:
Why do we use a custom redis image?

Reply:
because there was no upstream redis image available on quay or any other catalog than docker hub. And docker hub has rate limits... I don't know if the situation has changed

Reply:
So we can pull from quay.io, instead of docker (to avoid rate limiting) also described [here](https://github.com/operate-first/apps/blob/master/docs/argocd/update_argocd.md#updating-redis-image)